### PR TITLE
Gzip messages before base64 encoding

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Reduces the message payload by gzipping the cloud-info-provider output before encoding it and sending to AMS

---

**Related issue :** #144

### CODE REVIEW

**Comment by enolfc** [comment]
Not to be merged before AppDB implements the gzip support!

File: cloud-info/ams-wrapper.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ cloud-info-provider-service --yaml-file "$CLOUD_INFO_CONFIG" \
 ARGO_URL="https://$AMS_HOST/v1/projects/$AMS_PROJECT/topics/$AMS_TOPIC:publish?key=$AMS_TOKEN"
 
 printf '{"messages":[{"attributes":{},"data":"' > ams-payload
-grep -v "UNKNOWN" cloud-info.out | grep -v "^#" | base64 -w 0 >> ams-payload
+grep -v "UNKNOWN" cloud-info.out | grep -v "^#" | gzip | base64 -w 0 >> ams-payload
```
Comment (gwarf) [inline cloud-info/ams-wrapper.sh:38]: Just by curiosity, have you made some tests with gzipping the base64 vs gzipping the LDIF output? We could assume the LDIF may compress more, but may be interesting to check.

Comment (gwarf) [reply cloud-info/ams-wrapper.sh:38]: Cool, thanks for the confirmation.

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**